requirements/pytorch/extra.txt

```diff
@@ -8,3 +8,5 @@ hydra-core >=1.0.5, <1.4.0
 jsonargparse[signatures] >=4.18.0, <4.22.0
 rich >=12.3.0, <=13.0.1
 tensorboardX >=2.2, <=2.6  # min version is set by torch.onnx missing attribute
+torchdata >= 0.3.0
+s3fs >= 2023.5.0
```

Comment:
this is not compatible if you really need it, you need to update whole framework for this